uberenv.py

```diff
@@ -242,52 +242,73 @@ def parse_args():
                            " so it is disabled by default.")
 
     # option to stop after spack download and setup
-    parser.add_option("--setup-only",
+    parser.add_argument("--setup-only",
                       action="store_true",
                       dest="setup_only",
                       default=False,
-                      help="Only download and setup Spack. No further Spack command will be run.")
+                      help="Only download and setup the package manager. No further Spack command will be run. Will not create Spack Environment.")
 
     # option to skip spack download and setup
-    parser.add_option("--skip-setup",
+    parser.add_argument("--skip-setup",
                       action="store_true",
                       dest="skip_setup",
                       default=False,
                       help="Only install (using pre-setup Spack).")
 
+    # Spack Environment Setup
+    parser.add_argument("--spack-setup-environment",
```

Comment:
This is no longer used and should be removed.

Reply:
nm.. i searched for the wrong thing.. carry on.

But shouldn't it not be needed anymore because not providing a spack.yaml is implying this option.